Dockerfile

```diff
@@ -357,6 +341,29 @@ RUN chmod +x "${HOME}/run-tests-complete"
 
 RUN chmod +x "${HOME}/run-tests"
 
+# Copy selected subdirectories only
+COPY .github/ ${AIRFLOW_SOURCES}/.github/
+COPY dags/ ${AIRFLOW_SOURCES}/dags/
+COPY common/ ${AIRFLOW_SOURCES}/common/
+COPY licenses/ ${AIRFLOW_SOURCES}/licenses/
+COPY scripts/ci/in_container/ ${AIRFLOW_SOURCES}/scripts/ci/in_container/
+COPY docs/ ${AIRFLOW_SOURCES}/docs/
+COPY tests/ ${AIRFLOW_SOURCES}/tests/
+COPY airflow/ ${AIRFLOW_SOURCES}/airflow/
+COPY .coveragerc .rat-excludes .flake8 pylintrc LICENSE MANIFEST.in NOTICE CHANGELOG.txt \
+     .github .bash_completion .bash_completion.d run-tests run-tests-complete \
+     setup.cfg setup.py \
```

Comment:
Not that it makes much difference, but is ther a reson we are copying setup.py here again?  We already copied it in on L 311 (in this diff, line number has changed since.)

Reply:
(I was looking at something else and I was just curious. Since this file is so tiny it won't make much/any difference to speed)